.github/workflows/codeql-analysis.yml

```diff
@@ -32,14 +32,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'java', 'javascript' ]
+        language: ["java", "javascript"]
         # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
         # Learn more about CodeQL language support at https://git.io/codeql-language-support
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Setup JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: "17"
```

Comment:
Ik zou dit hetzelfde doen als in onze andere build workflow:

` java-version: ${{ env.JAVA_VERSION }}
`

en dan bovenin:

`
env:
  JAVA_VERSION: '17'
 `
 
 Maakt het net ietsje makkelijker om direct te zien welke Java versie we gebruiken. Detail hoor.